src/controllers/employees.js

```diff
@@ -1,129 +1,64 @@
-const fs = require('fs');
-const employees = require('../data/employees.json');
+import Employees from '../models/Employees';
 
-const getAllEmployees = (req, res) => {
-  res.status(200).json({ data: employees });
-};
-
-const getEmployeeById = (req, res) => {
-  const id = parseInt(req.params.id, 10);
-  const employee = employees.find((element) => element.id === id);
-  if (employee) {
-    res.status(200).json({ data: employee });
-  } else {
-    res.status(404).json({ error: 'Employee not found' });
-  }
-};
-
-const createNewEmployee = (req, res) => {
-  const id = parseInt(req.body.id, 10);
-  const employee = employees.find((element) => element.id === id);
-  const newEmployee = req.body;
-  const saveEmployee = {
-    id: newEmployee.id = Number(new Date().getTime().toString().substring(6)),
-    email: newEmployee.email,
-    password: newEmployee.password,
-    name: newEmployee.name,
-    lastName: newEmployee.lastName,
-    phone: newEmployee.phone,
-  };
-  if (JSON.stringify(newEmployee) === '{}') {
-    res.status(404).json({ error: 'Please provide employee data' });
-  } else if (employee) {
-    res.status(404).json({ error: 'Employee already exists' });
-  } else {
-    employees.push(saveEmployee);
-    fs.writeFile('./src/data/employees.json', JSON.stringify(employees), (err) => {
-      if (err) {
-        res.status(500).json({ error: 'Error writing file' });
-      } else {
-        res.status(201).json({ newEmployee });
-      }
+const getAllEmployees = async (req, res) => {
+  try {
+    const employees = await Employees.find();
```

Comment:
use query params here to filter the data.
```suggestion
    const employees = await Employees.find(req.query);
```